eth2util/keystore/keystore.go

```diff
@@ -62,6 +62,39 @@ func StoreKeys(secrets []*bls_sig.SecretKey, dir string) error {
 	return storeKeysInternal(secrets, dir, "keystore-%d.json")
 }
 
+// keymanagerReq represents the keymanager API request body for POST request. Refer: https://ethereum.github.io/keymanager-APIs/#/Local%20Key%20Manager/importKeystores
+type keymanagerReq struct {
+	Keystores          []keystore `json:"keystores"`
+	Passwords          []string   `json:"passwords"`
+	SlashingProtection string     `json:"slashing_protection"` // https://eips.ethereum.org/EIPS/eip-3076
```

Comment:
still need to figure out how to use it

Reply:
This fields is not relevant to `create cluster` as we are creating new validators which don't have a signing history as they aren't even activated yet.

**More on EIP-3076:**
A standard format for transferring a key’s signing history allows validators to easily switch between clients without the risk of signing conflicting messages. While a [common keystore format](https://eips.ethereum.org/EIPS/eip-2335) provides part of the solution, it does not contain any information about a key’s signing history. For a validator moving their keys from client A to client B, this could lead to scenarios in which client B inadvertently signs a message that conflicts with an earlier message signed with client A. The interchange format described here provides a solution to this problem.